modules/installation-configuration-parameters.adoc

```diff
@@ -1516,13 +1520,66 @@ ifdef::alibabacloud[]
 [id="installation-configuration-parameters-additional-alibaba_{context}"]
 == Additional Alibaba Cloud configuration parameters
 
-Additional Alibaba Cloud configuration parameters are described in the following table. The `alibabacloud` parameters are the configuration used when installing on Alibaba Cloud. The `defaultMachinePlatform` parameters are the default configuration used when installing on Alibaba Cloud for machine pools that do not define their own platform configuration.
+Additional Alibaba Cloud configuration parameters are described in the following table. 
+
+The `alibabacloud` parameters are the configuration used when installing on Alibaba Cloud. The `defaultMachinePlatform` parameters are the default configuration used when installing on Alibaba Cloud for machine pools that do not define their own platform configuration. 
+
+These parameters apply to both compute machines and control plane machines where specified.
+
+[NOTE]
+====
+If defined, the parameters `compute.platform.alibabacloud` and `controlPlane.platform.alibabacloud` will overwrite `platform.alibabacloud.defaultMachinePlatform` for compute machines and control plane machines respectively.
```

Comment:
The suggestion: 
If defined, the parameters under `compute.platform.alibabacloud` and `controlPlane.platform.alibabacloud` will overwrite `platform.alibabacloud.defaultMachinePlatform` settings for compute machines and control plane machines respectively.